tests/chainer_tests/test_gradient_check.py

```diff
@@ -142,6 +142,25 @@ def setUp(self):
         self.gys = (None,)
 
 
+class NumericalGradientReferenceTest(unittest.TestCase):
+
+    def setUp(self):
+        self.x = _uniform(2, 3)
+
+    def check_reference(self, x):
+        # A returned value and an input refers the same memory.
```

Comment:
I am afraid it would be difficult for readers to find out without context of issue #488 what this test want to check. One of the solution would be to refer to the issue.